aws/postgres/main.tf

```diff
@@ -35,7 +35,7 @@ resource "aws_vpc_security_group_egress_rule" "this" {
 
 module "main_db_instance" {
   source                       = "./db_instance"
-  identifier                   = local.identifier
+  identifier                   = try(var.db_identifier, local.identifier)
```

Comment:
should be `var. db_identifier != null ? var. db_identifier : local.identifier` @spartan-ytruong ?

Reply:
and please update in ` identifier           = replace(var.db_name, "_", "-")` instead

Reply:
The `local.identifier = replace(var.db_name, "_", "-")` is kept.
I add `var.db_identifier` here to flexibly allow the overwrite of identifier name for centralized terraform module migration rollout. I'm not sure if it may cause short downtime but it would help a lot during migration.